assets/js/atomic/blocks/index.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
```

Comment:
This file is imported by all products block to register the atomic blocks. These imports ensure all atomic blocks are loaded.